daemon/executor/Storage.java

```diff
@@ -187,15 +187,24 @@ public boolean isRunning() {
      * @throws GraknDaemonException
      */
     private void start() {
-        System.out.print("Starting " + DISPLAY_NAME + "...");
+        System.out.println("Starting " + DISPLAY_NAME + "...");
         System.out.flush();
 
         // Consume configuration from Grakn config file into Cassandra config file
         initialiseConfig();
 
         Future<Executor.Result> result = daemonExecutor.executeAsync(storageCommand(), graknHome.toFile());
 
-        LocalDateTime timeout = LocalDateTime.now().plusSeconds(STORAGE_STARTUP_TIMEOUT_SECOND);
+        // form storage timeout (default or ENV-value)
+        long startupTimeoutCalculated = STORAGE_STARTUP_TIMEOUT_SECOND;
+        String timeoutEnvValue = System.getenv("STORAGE_STARTUP_TIMEOUT_SECOND");
```

Comment:
1. We should declare the new config in `grakn.properties`:

```
# conf/grakn.properties
...
daemon.storage-bootup-timeout=180
```

2. We should declare the new config in https://github.com/graknlabs/grakn/blob/master/common/config/ConfigKey.java#L63

```
public static final ConfigKey<String> DAEMON_STORAGE_BOOTUP_TIMEOUT = key("daemon.storage-bootup-timeout", INT);
```

3. Finally, we should read the value from the config:

```
Integer timeoutEnvValue = graknProperties.getProperty(ConfigKey.DAEMON_STORAGE_BOOTUP_TIMEOUT)
```